freshmaker/__init__.py

```diff
@@ -43,6 +43,7 @@
 
 app = Flask(__name__)  # type: Any
 app.wsgi_app = ReverseProxy(app.wsgi_app)
+app.app_context().push()
```

Comment:
With the old version `flask-sqlalchemy`, we don't need to push the app context, right?

Reply:
True! That was slipping by, as well as two lines in `api_utils.py`. I reverted these changes, as they won't be needed anymore. Thanks for spotting!